how-to/debug-dependency-conflict.rst

```diff
@@ -47,7 +47,7 @@ same.
 
 ..  admonition:: But I didn't change anything in my project!
 
-    Because of the way pip works, even if you don't change anything at all in your project, simply
+    Because of the way pip works, even if you didn't change anything at all in your project, simply
```

Comment:
Was better previously, please revert this change.

Reply:
Done